### ISSUE TITLE
`extends` with `plugins` doesn't work

### ISSUE THREAD

**Issue by axten** [alexilyaev/stylelint-find-rules#18]
...so for now, removing this in favor of [stylelint-find-new-rules](https://github.com/Donov4n/stylelint-find-new-rules)

**Comment by alexilyaev** [comment]
@axten Is this related to https://github.com/alexilyaev/stylelint-find-rules/issues/9 specifically or something else is broken?

**Comment by axten** [comment]
I don't know...
What I can say is that is was broken with version 0.8
0.7 was working fine.

I just link to an order.js with rules,  like this
```
    extends: [
        './rules/order',
    ],
```

**Comment by alexilyaev** [comment]
@axten Can you please elaborate on your full use case?  
What did you expect the results to be?

in v0.8.0 I've added support for `extends`:  

> ## How does it handle extends?
> The rules of `extends` are added to the list of configured rules, thus, if a rule is covered in an extend config, it will not show up in the `unused` results.

So unless `./rules/order` has an `extends` of it's own, this should work.  
And by "work" I mean it should not show rules defined in the `extends` as being unused.

**Comment by alexilyaev** [comment]
@axten I've put in some work and now in v2.0.0 it fully supports extends of extends:  
https://github.com/alexilyaev/stylelint-find-rules/releases/tag/v2.0.0

Would you mind giving it a try and letting me know if it works for you?

**Comment by axten** [comment]
cool, it's not broken anymore!
`UNUSED` looks good

but `INVALID` is wrong:

```
INVALID: Configured rules that are no longer available:

RULE
order/order
order/properties-order
```
so 'stylelint-order' plugin is not considered correctly.

**Comment by alexilyaev** [comment]
Hmm, so it's time to support plugins :-)

**Comment by axten** [comment]
maybe https://github.com/Donov4n/stylelint-find-new-rules can give you a hint...
maybe its time to join forces

**Comment by alexilyaev** [comment]
Yeah, mentioned it at https://github.com/Donov4n/stylelint-find-new-rules/issues/2

And will check it as part of https://github.com/alexilyaev/stylelint-find-rules/issues/20
